test/rules/completed-docs/constructors/overloads/default/test.ts.lint

```diff
@@ -0,0 +1,31 @@
+class Foo {
```

Comment:
```suggestion
class Foo1 {
```

The TypeScript APIs might display some odd behavior around type inference with multiple classes of the same name. Since this file is checking edge cases of overloading, let's give each of these classes a unique name.

Reply:
_(on second though, a better name might be `FooPublic` or something that describes what the class is testing...)_

Reply:
Done